python/pyfunc-server/examples/echo_http/echo_http.py

```diff
@@ -0,0 +1,33 @@
+import logging
+
+import merlin
+from merlin.model import ModelType, PyFuncModel
+
+
+class EchoModel(PyFuncModel):
+    def initialize(self, artifacts):
+        pass
+
+    def infer(self, request):
+        logging.info("request: %s", request)
+        return request
+
+
+if __name__ == "__main__":
+    # Run pyfunc model locally without uploading to Merlin server
+    merlin.run_pyfunc_model(
+        model_instance=EchoModel(),
+        conda_env="env.yaml",
+    )
+
+    # Or, if you already have logged existing model version on Merlin,
+    # you can get the latest model version and run it locally:
+    # merlin.set_url("<your merlin server url>")
+    # merlin.set_project("<your project>")
+    # merlin.set_model("<your model name>", ModelType.PYFUNC)
+
+    # versions = merlin.active_model().list_version()
+    # versions.sort(key=lambda v: v.id, reverse=True)
+
+    # last_version = versions[0]
+    # last_version.start_server(debug=True)
```

Comment:
this is bit odd for me. If i want to run a local pyfunc,
```
merlin.run_pyfunc_model(
  model_instance=...
  artifact=...
```
but if i want to run latest version from remote mlflow, it is call `start_server`, can this be rename to `run_pyfunc_model` as well?

Reply:
Agree that we can simplify this and need more time to think about the flow. One thing that I consider is to have `start_server()` in fluent.py so we can call it via `merlin.start_server()` which will start the active model version's local server. But this flow is only good for new model_version, which again will create a "record" in both merlin and mlflow storage.

Currently, the `merlin.deploy` function already has `deployment_mode` argument to deploy the model as serverless or raw deployment, so we can use another argument here (i.e. `run_local`). However, I think it still better to have one function to do one thing. Deploy should just be used to deploy, and to run locally, it should be using different function.

Reply:
hmm in this case can we hide `start_server` from the examples? 

at least we only show/advocate using`merlin.run_pyfunc_model()` and `merlin.deploy()` which is more consistent